docs/analytics-gtm/ga4-link-tracker.md

```diff
@@ -25,10 +28,10 @@ In the example above, on a left click of the link, the following would be pushed
 
 ```
 {
-    "event": "analytics",
+    "event": "event_data",
     "event_data": {
```

Comment:
Probably out of scope of the code change, but it seems odd that the value of `event` and another key are both `event_data`. Is this correct?

Reply:
I think it's correct as it was requested by our Performance Analyst - I'm guessing there's a good reason for this (maybe they need the event key in order to determine which object they need to grab and process, since they never know what type of object is being pushed to the dataLayer?)